wp-content/plugins/core/src/Query/Search.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Query;
+
+class Search {
+
+	/**
+	 * Prevents the query for the WP Core search from running when there is no search term `?s=`,
+	 * but still returns the search route (page title, template, etc).
+	 *
+	 * @param           $request
+	 * @param \WP_Query $query
+	 *
+	 * @return bool
+	 *
+	 * @hook https://developer.wordpress.org/reference/hooks/posts_request/
+	 */
+	public function dont_do_search( $request, \WP_Query $query ) {
```

Comment:
Can we rename this to something like `force_load_search_template()`?

Reply:
Good idea!